src/read.c

```diff
@@ -3800,7 +3795,6 @@ avifResult avifDecoderReset(avifDecoder * decoder)
         return AVIF_RESULT_BMFF_PARSE_FAILED;
     }
 
-    avifDecoderFlush(decoder);
```

Comment:
Can this `avifDecoderFlush()` call at the end of `avifDecoderReset()` be omitted?

Reply:
I was asking why it is correct to remove the `avifDecoderFlush()` call at the end of `avifDecoderReset()`. I see this was suggested by Yannis and he explained why this `avifDecoderFlush()` call can be removed.

Reply:
Yes, essentially this is okay because we call `avifDecoderDataClearTiles` in line 3416 and that does more than what `avifDecoderFlush` used to do.

Reply:
One complication is that after we call `avifDecoderDataClearTiles` in line 3416, we call `avifDecoderDataCreateTile`, which creates `tile->image` as empty images. So we need to verify that `tile->image` is still empty at the end of `avifDecoderReset`. I validated that by adding some assertions and running our test suite, but this is also very plausible because we don't have any `tile->codec` to decode data into `tile->image`.

This analysis shows that more work could be moved out of `avifDecoderParse` to optimize for the "parse container metadata only" use case. For example, the empty `tile->image` that we create is not used in this use case.